recipes/ibm-watson/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "ibm-watson" %}
+{% set version = "4.4.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b3700014e13aa5a211b22837a1a370fb24f08ff6d27935db1c19fa901e9c423c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - ibm-cloud-sdk-core ==1.5.1
```

Comment:
does this need to be pinned so tightly?

Reply:
This is just what grayskull spit out. I'm not actually all that familiar with these packages, but a long time ago I took on a co-maintainer role on `opsdroid` (can't remember why since I don't use it. I think I was just just helping out since another packaged needed it on staged-recipes at the time). opsdroid has fallen way out of date and I was just hoping to bring it up to the current version.

I could make the version restriction more permissive. In this case would `>=1.5,<1.6` be reasonable?

Reply:
that would be good

Reply:
Ok, did that. Also is all of the `-` vs `_` in the naming ok? I think I tried to follow the pip convention where it was an issue.